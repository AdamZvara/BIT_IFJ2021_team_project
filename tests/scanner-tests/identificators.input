id      ID

id1 
i1d   

i_d
_id
id__

id_1
_1id
_id1

iiiiiiiiiiiiiiiiiiiiiiiiiidddddddddddddddddddddddd