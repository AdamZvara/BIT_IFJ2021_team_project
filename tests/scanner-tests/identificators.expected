TOK_ID : id
TOK_ID : ID
TOK_ID : id1
TOK_ID : i1d
TOK_ID : i_d
TOK_ID : _id
TOK_ID : id__
TOK_ID : id_1
TOK_ID : _1id
TOK_ID : _id1
TOK_ID : iiiiiiiiiiiiiiiiiiiiiiiiiidddddddddddddddddddddddd
